cmake/SetupProtobuf.cmake

```diff
@@ -24,9 +24,15 @@ function(_userver_set_protobuf_version_category)
   endif()
 endfunction()
 
+function(_userver_set_protobuf_variables)
```

Comment:
It will be simpler to track down `GENERATE_PROTOS_AT_CONFIGURE_DEFAULT` value if we inline and remove this function